### PR TITLE
Referenced latest mx-chain-go version

### PR DESCRIPTION
**Pull request by iulianpascalau**
- referenced latest mx-chain-go version

### CODE REVIEW

File: cmd/chainsimulator/config/nodeOverrideDefault.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# System overrides, DO NOT EDIT THIS FILE
+OverridableConfigTomlValues = [
+    { File = "systemSmartContractsConfig.toml", Path = "StakingSystemSCConfig.NodeLimitPercentage", Value = "1.0" }, # no staking limits imposed
+    # TODO remove the following overrides when integrating a newer version of the node (greater than v1.7.10)
+    { File = "systemSmartContractsConfig.toml", Path = "DelegationManagerSystemSCConfig.MinStakeAmount", Value = "1000000000000000000" }, # 1 EGLD as on mainnet
+]
```
Comment (miiu96) [inline cmd/chainsimulator/config/nodeOverrideDefault.toml:6]: add empty line at the end of the file

File: config/testdata/override1.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+OverridableConfigTomlValues = [
+    { File = "systemSmartContractsConfig.toml", Path = "StakingSystemSCConfig.NodeLimitPercentage", Value = "1.0" }, # no staking limits imposed
+]
```
Comment (miiu96) [inline config/testdata/override1.toml:3]: same here

File: config/testdata/override2.toml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+OverridableConfigTomlValues = [
+    { File = "config.toml", Path = "MiniBlocksStorage.Cache.Name", Value = "MiniBlocksStorage" },
+    { File = "external.toml", Path = "ElasticSearchConnector.Enabled", Value = "true" }
+]
```
Comment (miiu96) [inline config/testdata/override2.toml:4]: also here

File: config/testdata/override3.toml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+OverridableConfigTomlValues = [
+    { File = "external.toml", Path = "ElasticSearchConnector.Enabled", Value = "false" }
+]
```
Comment (miiu96) [inline config/testdata/override3.toml:3]: also here

**Review by sstanculeanu — COMMENTED** [review]
no other comments

File: cmd/chainsimulator/config/nodeOverrideDefault.toml (hunk 1 above)
Comment (iulianpascalau) [reply cmd/chainsimulator/config/nodeOverrideDefault.toml:6]: done

File: config/testdata/override1.toml (hunk 2 above)
Comment (iulianpascalau) [reply config/testdata/override1.toml:3]: done

File: config/testdata/override2.toml (hunk 3 above)
Comment (iulianpascalau) [reply config/testdata/override2.toml:4]: done

File: config/testdata/override3.toml (hunk 4 above)
Comment (iulianpascalau) [reply config/testdata/override3.toml:3]: done

**Review by sstanculeanu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**